### PR TITLE
Remove Avalanche Wallet references

### PR DESCRIPTION
**Pull request by laviniat1996**

### CODE REVIEW

**Comment by yulin-dong** [comment]
I will review when I can.

File: docs/build/dapp/smart-contracts/remix-deploy.md [hunk 1]
```diff
@@ -49,13 +49,16 @@ To switch to the **Fuji test network**, go to Settings, select Advanced, and the
 
 ## Step 2: Funding Your C-Chain Address
 
-### **Using Avalanche Wallet**
+### **Using Core web**
```
Comment (ashucoder9) [inline docs/build/dapp/smart-contracts/remix-deploy.md:52]:
```suggestion
### **Using Core Web**
```

File: docs/build/dapp/smart-contracts/remix-deploy.md [hunk 2]
```diff
@@ -49,13 +49,16 @@ To switch to the **Fuji test network**, go to Settings, select Advanced, and the
...
-On the Mainnet, you can use the [Avalanche
-Wallet](https://wallet.avax.network/) to transfer funds from the X-Chain to your
+On the Mainnet, you can use [Core
+web](https://core.app/) to transfer funds from the X-Chain to your
 C-Chain address. The process is simple, as explained in this
-[tutorial](https://support.avax.network/en/articles/6169872-how-to-make-a-cross-chain-transfer-in-the-avalanche-wallet).
-Wallet can be used on test and local networks, too.
+[tutorial](https://support.avax.network/en/articles/8133713-core-web-how-do-i-make-cross-chain-transfers-in-core-stake).
+Please note that you will need [Core wallet](https://join.core.app/extension) connected to Core web for making cross-chain transfers.
+Core wallet can be used on test and local networks, too.
```
Comment (ashucoder9) [inline docs/build/dapp/smart-contracts/remix-deploy.md:59]:
```suggestion
The Core wallet can be used on test and local networks, and you can download it on [mobile](https://support.avax.network/en/articles/6115608-core-mobile-where-can-i-download-core-mobile-to-my-phone) too.
```

File: docs/build/dapp/smart-contracts/remix-deploy.md [hunk 3]
```diff
@@ -49,13 +49,16 @@ To switch to the **Fuji test network**, go to Settings, select Advanced, and the
...
-Wallet](https://wallet.avax.network/) to transfer funds from the X-Chain to your
+On the Mainnet, you can use [Core
+web](https://core.app/) to transfer funds from the X-Chain to your
 C-Chain address. The process is simple, as explained in this
-[tutorial](https://support.avax.network/en/articles/6169872-how-to-make-a-cross-chain-transfer-in-the-avalanche-wallet).
-Wallet can be used on test and local networks, too.
+[tutorial](https://support.avax.network/en/articles/8133713-core-web-how-do-i-make-cross-chain-transfers-in-core-stake).
+Please note that you will need [Core wallet](https://join.core.app/extension) connected to Core web for making cross-chain transfers.
+Core wallet can be used on test and local networks, too. 
+This wallet is available for
```
Comment (ashucoder9) [inline docs/build/dapp/smart-contracts/remix-deploy.md:60]:
```suggestion
```

File: docs/build/dapp/smart-contracts/remix-deploy.md [hunk 4]
```diff
@@ -49,13 +49,16 @@ To switch to the **Fuji test network**, go to Settings, select Advanced, and the
...
+On the Mainnet, you can use [Core
+web](https://core.app/) to transfer funds from the X-Chain to your
 C-Chain address. The process is simple, as explained in this
-[tutorial](https://support.avax.network/en/articles/6169872-how-to-make-a-cross-chain-transfer-in-the-avalanche-wallet).
-Wallet can be used on test and local networks, too.
+[tutorial](https://support.avax.network/en/articles/8133713-core-web-how-do-i-make-cross-chain-transfers-in-core-stake).
+Please note that you will need [Core wallet](https://join.core.app/extension) connected to Core web for making cross-chain transfers.
+Core wallet can be used on test and local networks, too. 
+This wallet is available for 
+[mobile](https://support.avax.network/en/articles/6115608-core-mobile-where-can-i-download-core-mobile-to-my-phone) too.
```
Comment (ashucoder9) [inline docs/build/dapp/smart-contracts/remix-deploy.md:61]:
```suggestion
```

File: docs/build/subnet/deploy/fuji-testnet-subnet.md [hunk 5]
```diff
@@ -218,12 +218,13 @@ the use of a ledger device is strongly recommended.
...
-   "Private key" while accessing the [web wallet](https://wallet.avax.network). (Private Key is the
-   first option on the [web wallet](https://wallet.avax.network)).
-3. Move the test funds from the C-Chain to the P-Chain by clicking on the `Cross Chain` on the left
-   side of the web wallet (find more details on
-   [this tutorial](https://support.avax.network/en/articles/6169872-how-to-make-a-cross-chain-transfer-in-the-avalanche-wallet)).
+2. **Export** your key via the `avalanche key export` command. The output is your private key,
+   which you will [import](https://support.avax.network/en/articles/6821877-core-extension-how-can-i-import-an-account)
+   to Core extension.
+
+3. Connect Core extension to [Core web](https://core.app/), and move the test funds from the C-Chain to the P-Chain
```
Comment (ashucoder9) [inline docs/build/subnet/deploy/fuji-testnet-subnet.md:225]:
```suggestion
3. Connect the Core extension to [Core Web](https://core.app/), and move the test funds from C-Chain to the P-Chain
```

File: docs/build/subnet/deploy/fuji-testnet-subnet.md [hunk 6]
```diff
@@ -218,12 +218,13 @@ the use of a ledger device is strongly recommended.
...
    [`https://faucet.avax.network`](https://faucet.avax.network/) using your **C-Chain address**.
 
-2. **Export** your key via the `avalanche key export` command, then paste the output when selecting
-   "Private key" while accessing the [web wallet](https://wallet.avax.network). (Private Key is the
-   first option on the [web wallet](https://wallet.avax.network)).
-3. Move the test funds from the C-Chain to the P-Chain by clicking on the `Cross Chain` on the left
-   side of the web wallet (find more details on
-   [this tutorial](https://support.avax.network/en/articles/6169872-how-to-make-a-cross-chain-transfer-in-the-avalanche-wallet)).
+2. **Export** your key via the `avalanche key export` command. The output is your private key,
+   which you will [import](https://support.avax.network/en/articles/6821877-core-extension-how-can-i-import-an-account)
```
Comment (ashucoder9) [inline docs/build/subnet/deploy/fuji-testnet-subnet.md:222]:
```suggestion
   which will help you [import](https://support.avax.network/en/articles/6821877-core-extension-how-can-i-import-an-account) your account
```

File: docs/build/subnet/deploy/fuji-testnet-subnet.md [hunk 7]
```diff
@@ -218,12 +218,13 @@ the use of a ledger device is strongly recommended.
...
+   to Core extension.
```
Comment (ashucoder9) [inline docs/build/subnet/deploy/fuji-testnet-subnet.md:223]:
```suggestion
into the Core extension.
```

File: docs/build/subnet/deploy/fuji-testnet-subnet.md [hunk 8]
```diff
@@ -218,12 +218,13 @@ the use of a ledger device is strongly recommended.
...
+
+3. Connect Core extension to [Core web](https://core.app/), and move the test funds from the C-Chain to the P-Chain
+   by clicking Stake, then Cross-Chain Transfer ( find more details on
```
Comment (ashucoder9) [inline docs/build/subnet/deploy/fuji-testnet-subnet.md:226]:
```suggestion
   by clicking Stake, then Cross-Chain Transfer (find more details on
```

File: docs/build/subnet/deploy/fuji-testnet-subnet.md [hunk 9]
```diff
@@ -218,12 +218,13 @@ the use of a ledger device is strongly recommended.
...
+   [this tutorial](https://support.avax.network/en/articles/8133713-core-web-how-do-i-make-cross-chain-transfers-in-core-stake) ).
```
Comment (ashucoder9) [inline docs/build/subnet/deploy/fuji-testnet-subnet.md:227]:
```suggestion
   [this tutorial](https://support.avax.network/en/articles/8133713-core-web-how-do-i-make-cross-chain-transfers-in-core-stake)).
```

File: docs/build/subnet/deploy/mainnet-subnet.md [hunk 10]
```diff
@@ -133,14 +133,14 @@ with the `--local` parameter.
...
-You can load the Ledger's C-Chain address in the web wallet or load in a different private key. You
-can transfer funds from the C-Chain to the P-Chain by clicking on the Cross Chain on the left side
-of the web wallet. See this
-[tutorial](https://support.avax.network/en/articles/6169872-how-to-make-a-cross-chain-transfer-in-the-avalanche-wallet-between-x-and-c-chain)
-for more instructions.
+exporting them from the C-Chain to the P-Chain using [Core web](https://core.app)
+connected to [Core extension](https://join.core.app/extension).
+
+You can load the Ledger's C-Chain address in Core extension, or load in a different private key to 
+[Core extension](https://join.core.app/extension), then connect to Core web .
```
Comment (ashucoder9) [inline docs/build/subnet/deploy/mainnet-subnet.md:140]:
```suggestion
[Core extension](https://join.core.app/extension), and then connect to Core Web. 
```

File: docs/build/subnet/deploy/mainnet-subnet.md [hunk 11]
```diff
@@ -133,14 +133,14 @@ with the `--local` parameter.
...
-of the web wallet. See this
-[tutorial](https://support.avax.network/en/articles/6169872-how-to-make-a-cross-chain-transfer-in-the-avalanche-wallet-between-x-and-c-chain)
-for more instructions.
+exporting them from the C-Chain to the P-Chain using [Core web](https://core.app)
+connected to [Core extension](https://join.core.app/extension).
+
+You can load the Ledger's C-Chain address in Core extension, or load in a different private key to 
+[Core extension](https://join.core.app/extension), then connect to Core web . 
+
+You can move test funds from the C-Chain to the P-Chain by clicking Stake on Core web, then Cross-Chain Transfer ( find more details on
```
Comment (ashucoder9) [inline docs/build/subnet/deploy/mainnet-subnet.md:142]:
```suggestion
You can move test funds from C-Chain to P-Chain by clicking Stake on Core Web, then Cross-Chain Transfer (find more details on
```

File: docs/build/subnet/deploy/mainnet-subnet.md [hunk 12]
```diff
@@ -133,14 +133,14 @@ with the `--local` parameter.
...
+[this tutorial](https://support.avax.network/en/articles/8133713-core-web-how-do-i-make-cross-chain-transfers-in-core-stake) ).
```
Comment (ashucoder9) [inline docs/build/subnet/deploy/mainnet-subnet.md:143]:
```suggestion
[this tutorial](https://support.avax.network/en/articles/8133713-core-web-how-do-i-make-cross-chain-transfers-in-core-stake)).
```

**Review by ashucoder9 — CHANGES_REQUESTED** [review]
Thank you @laviniat1996, suggested some edits 🙏🏽

File: docs/nodes/run/third-party/aws-marketplace-one-click.md [hunk 13]
```diff
@@ -272,89 +272,118 @@ the `nodeID` is `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
 }
 ```
 
-## Add Node as Validator on Fuji via the Web Wallet
+## Add Node as Validator on Fuji via Core web
```
Comment (ashucoder9) [inline docs/nodes/run/third-party/aws-marketplace-one-click.md:275]:
```suggestion
## Add Node as Validator on Fuji via Core Web
```

File: docs/nodes/run/third-party/aws-marketplace-one-click.md [hunk 14]
```diff
@@ -272,89 +272,118 @@ the `nodeID` is `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
...
 
 For adding the new node as a Validator on the Fuji testnet's Primary Network you can
-use the [Avalanche Web Wallet](https://wallet.avax.network).
+use the [Core web](https://core.app/)
```
Comment (ashucoder9) [inline docs/nodes/run/third-party/aws-marketplace-one-click.md:278]:
```suggestion
use the [Core Web](https://core.app/) 
```

File: docs/nodes/run/third-party/aws-marketplace-one-click.md [hunk 15]
```diff
@@ -272,89 +272,118 @@ the `nodeID` is `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
...
 
-![Avalanche Web Wallet](/img/one-click-validator-node/web-wallet.png)
+![Core web](/img/one-click-validator-node/core-web.png)
 
-The Avalanche Web Wallet is a web-based application with no middleware or any
-kind of server communication. It can be either accessed
-online or compiled and run locally. The Avalanche Web Wallet is a multi-faceted
-jewel and offers validation/delegation, cross-chain transfers, reward
-estimation, asset/key management, and more.
+Core web is a free, all-in-one command center giving users a more intuitive and comprehensive way to view,
```
Comment (ashucoder9) [inline docs/nodes/run/third-party/aws-marketplace-one-click.md:288]:
```suggestion
Core Web is a free, all-in-one command center that gives users a more intuitive and comprehensive way to view,
```

File: docs/nodes/run/third-party/aws-marketplace-one-click.md [hunk 16]
```diff
@@ -272,89 +272,118 @@ the `nodeID` is `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
...
+and use across the Avalanche network, Avalanche Subnets, and Ethereum.
```
Comment (ashucoder9) [inline docs/nodes/run/third-party/aws-marketplace-one-click.md:289]:
```suggestion
and use dApps across the Avalanche network, its various Subnets, and Ethereum. 
```

File: docs/nodes/run/third-party/aws-marketplace-one-click.md [hunk 17]
```diff
@@ -272,89 +272,118 @@ the `nodeID` is `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
...
+![Core web](/img/one-click-validator-node/core-web.png)
 
-The Avalanche Web Wallet is a web-based application with no middleware or any
-kind of server communication. It can be either accessed
-online or compiled and run locally. The Avalanche Web Wallet is a multi-faceted
-jewel and offers validation/delegation, cross-chain transfers, reward
-estimation, asset/key management, and more.
+Core web is a free, all-in-one command center giving users a more intuitive and comprehensive way to view,
+and use across the Avalanche network, Avalanche Subnets, and Ethereum. 
+Core web is optimized for use with Core browser extension and Core mobile (iOS, Android), free,
```
Comment (ashucoder9) [inline docs/nodes/run/third-party/aws-marketplace-one-click.md:290]:
```suggestion
Core Web is optimized for use with the Core browser extension and Core mobile (available on both iOS & Android).
```

File: docs/nodes/run/third-party/aws-marketplace-one-click.md [hunk 18]
```diff
@@ -272,89 +272,118 @@ the `nodeID` is `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
...
 
-The Avalanche Web Wallet is a web-based application with no middleware or any
-kind of server communication. It can be either accessed
-online or compiled and run locally. The Avalanche Web Wallet is a multi-faceted
-jewel and offers validation/delegation, cross-chain transfers, reward
-estimation, asset/key management, and more.
+Core web is a free, all-in-one command center giving users a more intuitive and comprehensive way to view,
+and use across the Avalanche network, Avalanche Subnets, and Ethereum. 
+Core web is optimized for use with Core browser extension and Core mobile (iOS, Android), free, 
+personal wallet (self-custodial). Together, they are key components of the Core product
```
Comment (ashucoder9) [inline docs/nodes/run/third-party/aws-marketplace-one-click.md:291]:
```suggestion
Together, they are key components of the Core product 
```

File: docs/nodes/run/third-party/aws-marketplace-one-click.md [hunk 19]
```diff
@@ -272,89 +272,118 @@ the `nodeID` is `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
...
+Core web is optimized for use with Core browser extension and Core mobile (iOS, Android), free, 
+personal wallet (self-custodial). Together, they are key components of the Core product 
+suite that brings dApps, NFTs, Avalanche Bridge, Subnets, L2s, and more, directly to users.
 
-### Switching the Connected Network
+### Switching to Testnet Mode
 
-Check which network the wallet is connected to by looking at the top right of
-the screen. By default the Avalanche Web Wallet connects to Mainnet.
+By default, Core web and Core extension are connected to Mainnet.
```
Comment (ashucoder9) [inline docs/nodes/run/third-party/aws-marketplace-one-click.md:296]:
```suggestion
By default, Core Web and Core extension are connected to Mainnet. 
```

File: docs/nodes/run/third-party/aws-marketplace-one-click.md [hunk 20]
```diff
@@ -272,89 +272,118 @@ the `nodeID` is `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
...
+amount is 1 AVAAX.
+* Delegation Fee: You will claim this % of the rewards from the delegators on
+your node.
+* Reward Address: A reward address is the destination address of the accumulated
+staking rewards.
+:::
 
-<img src="/img/one-click-validator-node/network-fuji.png" alt="Network - Fuji"
-width="60%" />
+To add a node as a Validator, first select the Stake tab on Core web,
```
Comment (ashucoder9) [inline docs/nodes/run/third-party/aws-marketplace-one-click.md:331]:
```suggestion
To add a node as a Validator, first select the Stake tab on Core Web, 
```

File: docs/nodes/validate/add-a-validator.md [hunk 21]
```diff
@@ -46,9 +46,9 @@ when executing the AvalancheGo binary, as by default, the node will attempt to p
 NAT traversal to get the node's IP according to its router). Failing to do either of
 these may jeopardize your staking reward.
 
-## Add a Validator with Avalanche Wallet
+## Add a Validator with Core extension
 
-First, we show you how to add your node as a validator by using [Avalanche Wallet](https://wallet.avax.network).
+First, we show you how to add your node as a validator by using [Core web](https://core.app).
```
Comment (ashucoder9) [inline docs/nodes/validate/add-a-validator.md:51]:
```suggestion
First, we show you how to add your node as a validator by using [Core Web](https://core.app).
```

File: docs/nodes/validate/add-a-validator.md [hunk 22]
```diff
@@ -76,28 +76,30 @@ The response has your node’s ID:
 
 ### Add as a Validator
 
-Open [the wallet](https://wallet.avax.network/), and go the `Earn` tab. Choose
-`Add Validator` under the `Validate` section.
+Connect [Core extension](https://join.core.app/extension) to [Core web](https://core.app), and go the 'Staking' tab.
```
Comment (ashucoder9) [inline docs/nodes/validate/add-a-validator.md:79]:
```suggestion
Connect [Core extension](https://join.core.app/extension) to [Core Web](https://core.app), and go to the 'Staking' tab. 
```

File: docs/nodes/validate/add-a-validator.md [hunk 23]
```diff
@@ -76,28 +76,30 @@ The response has your node’s ID:
 
 ### Add as a Validator
 
-Open [the wallet](https://wallet.avax.network/), and go the `Earn` tab. Choose
-`Add Validator` under the `Validate` section.
+Connect [Core extension](https://join.core.app/extension) to [Core web](https://core.app), and go the 'Staking' tab. 
+Here, choose`Validate` from the menu.
```
Comment (ashucoder9) [inline docs/nodes/validate/add-a-validator.md:80]:
```suggestion
Here, choose 'Validate` from the menu.
```

File: docs/reference/standards/cryptographic-primitives.md [hunk 24]
```diff
@@ -184,10 +184,14 @@ As an example we will sign the message "Through consensus to the stars"
 After hashing with `sha256` and signing the pre-image we return the value
 [cb58](https://support.avalabs.org/en/articles/4587395-what-is-cb58) encoded:
 `4Eb2zAHF4JjZFJmp4usSokTGqq9mEGwVMY2WZzzCmu657SNFZhndsiS8TvL32n3bexd8emUwiXs8XqKjhqzvoRFvghnvSN`.
-Here's an example using the [Avalanche Web
-Wallet](https://wallet.avax.network/wallet/advanced).
+Here's an example using [Core web](https://core.app/tools/signing-tools/sign/).
 
-![Sign message](/img/sign-message.png)
+:::info
+A full guide on how to sign messages with Core web can be found
```
Comment (ashucoder9) [inline docs/reference/standards/cryptographic-primitives.md:190]:
```suggestion
A full guide on how to sign messages with Core Web can be found 
```

**Review by ashucoder9 — CHANGES_REQUESTED** [review]
Suggestion: you may mark this PR as a draft @laviniat1996 until it's ready for a final review.

File: docs/build/dapp/smart-contracts/remix-deploy.md (hunk 1 above)
Comment (laviniat1996) [reply docs/build/dapp/smart-contracts/remix-deploy.md:52]: @ashucoder9 That's a valid suggestion, I was conflicted on this one too. We use 'Core web' in our support docs https://support.avax.network/en/?q=core+web . I thought I should keep the same pattern here. What do you think I should do?

File: docs/build/dapp/smart-contracts/remix-deploy.md (hunk 2 above)
Comment (laviniat1996) [reply docs/build/dapp/smart-contracts/remix-deploy.md:59]: fixed aee59acc5055056042b9fc36caa38ebce92421d5

File: docs/build/subnet/deploy/fuji-testnet-subnet.md (hunk 6 above)
Comment (laviniat1996) [reply docs/build/subnet/deploy/fuji-testnet-subnet.md:222]: fixed aee59acc5055056042b9fc36caa38ebce92421d5

File: docs/build/subnet/deploy/fuji-testnet-subnet.md (hunk 9 above)
Comment (laviniat1996) [reply docs/build/subnet/deploy/fuji-testnet-subnet.md]: fixed eb9c4fb7554477079d0b256e5e263d1d91435443

File: docs/build/subnet/deploy/fuji-testnet-subnet.md (hunk 8 above)
Comment (laviniat1996) [reply docs/build/subnet/deploy/fuji-testnet-subnet.md]: fixed eb9c4fb7554477079d0b256e5e263d1d91435443

File: docs/build/subnet/deploy/fuji-testnet-subnet.md (hunk 5 above)
Comment (laviniat1996) [reply docs/build/subnet/deploy/fuji-testnet-subnet.md]: fixed eb9c4fb7554477079d0b256e5e263d1d91435443

File: docs/build/subnet/deploy/fuji-testnet-subnet.md (hunk 7 above)
Comment (laviniat1996) [reply docs/build/subnet/deploy/fuji-testnet-subnet.md]: fixed eb9c4fb7554477079d0b256e5e263d1d91435443

File: docs/build/subnet/deploy/fuji-testnet-subnet.md (hunk 6 above)
Comment (laviniat1996) [reply docs/build/subnet/deploy/fuji-testnet-subnet.md]: fixed eb9c4fb7554477079d0b256e5e263d1d91435443

File: docs/build/subnet/deploy/mainnet-subnet.md [hunk 25]
```diff
@@ -133,14 +133,14 @@ with the `--local` parameter.
...
 A new Ledger device has no funds on the addresses it controls. You'll need to send funds to it by
-exporting them from the C-Chain to the P-Chain using [Avalanche's Web
-Wallet](https://wallet.avax.network).
-
-You can load the Ledger's C-Chain address in the web wallet or load in a different private key. You
-can transfer funds from the C-Chain to the P-Chain by clicking on the Cross Chain on the left side
-of the web wallet. See this
-[tutorial](https://support.avax.network/en/articles/6169872-how-to-make-a-cross-chain-transfer-in-the-avalanche-wallet-between-x-and-c-chain)
-for more instructions.
+exporting them from the C-Chain to the P-Chain using [Core web](https://core.app)
```
[the remark this answers is not in the document]
Comment (laviniat1996) [reply docs/build/subnet/deploy/mainnet-subnet.md]: fixed 8033a0c4a32eae85925c0dc9eb6c4674ee616781

File: docs/nodes/run/third-party/aws-marketplace-one-click.md (hunk 15 above)
Comment (laviniat1996) [reply docs/nodes/run/third-party/aws-marketplace-one-click.md]: fixed 798fcbef0fd3855c86d950a2af92d9b3ba43f1a4

File: docs/nodes/run/third-party/aws-marketplace-one-click.md (hunk 16 above)
Comment (laviniat1996) [reply docs/nodes/run/third-party/aws-marketplace-one-click.md]: fixed 798fcbef0fd3855c86d950a2af92d9b3ba43f1a4

File: docs/nodes/run/third-party/aws-marketplace-one-click.md (hunk 18 above)
Comment (laviniat1996) [reply docs/nodes/run/third-party/aws-marketplace-one-click.md]: fixed https://support.avax.network/en/?q=core+mobile

**Comment by laviniat1996** [comment]
@ashucoder9 The PR is ready now (except is not translated to spanish). I think I've addressed all of your comments. We still have to choose if we use capital letters for 'mobile' and 'web'.

**Comment by ashucoder9** [comment]
/gt docs/build/dapp/fuji-workflow.md i18n/es/docusaurus-plugin-content-docs/current/build/dapp/fuji-workflow.md spanish

**Comment by ashucoder9** [comment]
/gt docs/build/dapp/smart-contracts/remix-deploy.md i18n/es/docusaurus-plugin-content-docs/current/build/dapp/smart-contracts/remix-deploy.md spanish

**Comment by ashucoder9** [comment]
/gt docs/build/subnet/deploy/fuji-testnet-subnet.md i18n/es/docusaurus-plugin-content-docs/current/build/subnet/deploy/fuji-testnet-subnet.md spanish

**Comment by ashucoder9** [comment]
/gt docs/nodes/validate/add-a-validator.md i18n/es/docusaurus-plugin-content-docs/current/nodes/validate/add-a-validator.md spanish

**Comment by ashucoder9** [comment]
/gt docs/build/subnet/deploy/fuji-testnet-subnet.md i18n/es/docusaurus-plugin-content-docs/current/build/subnet/deploy/fuji-testnet-subnet.md spanish

**Pull request closed without merging**